patches/800_delete_phm_templates_and_insert_cris_templates.sql

```diff
@@ -0,0 +1,30 @@
+-- ****************************************************************************
+-- RM SQL DATABASE DELETE AND INSERT SCRIPT
+-- ****************************************************************************
+-- Number: 800
+-- Purpose: Delete PHM pack codes and insert CRIS pack codes
+-- Author: Liam Toozer
+-- ****************************************************************************
+
+DELETE FROM casev3.export_file_template
+WHERE pack_code IN ('NL_EN_ST_PHM', 'NL_CY_ST_PHM', 'NL_EN_LP_PHM', 'NL_CY_LP_PHM');
+
+DELETE FROM casev3.email_template
+WHERE pack_code IN ('NE_EN_Q_PHM', 'NE_CY_Q_PHM', 'NE_EN_T_PHM', 'NE_CY_T_PHM', 'RE_EN_PHM', 'RE_CY_PHM', 'UR_EN_PHM', 'UR_CY_PHM');
```

Comment:
These `DELETE`s will run into problems in environments which are being used, there are foreign key references to the templates from the `action_rule`, `..._to_process`, and survey/template linking tables. I think this will need to be handled more carefully than just a delete in the update script.